src/llm_vm/onsite_llm.py

```diff
@@ -299,9 +299,9 @@ class Small_Local_LLama(Base_Onsite_LLM):
     model_uri="openlm-research/open_llama_3b_v2"
```

Comment:
this is the URL you need to fix

Reply:
Updated the requested URL to Llama-2-70b-hf
Source:https://huggingface.co/blog/llama2 (https://huggingface.co/meta-llama/Llama-2-70b-hf)

Reply:
What I mean is: defaults for models should always be the smallest version possible so anyone can run the code locally